common/src/main/scala/uk/ac/wellcome/models/IdentityState.scala

```diff
@@ -0,0 +1,23 @@
+package uk.ac.wellcome.models
+
+sealed trait IdentityState[+T]
+
+sealed trait IdentifieableOrUnidentifiable[+T] extends IdentityState[T]
+sealed trait IdentifiedOrUnidentifiable[+T] extends IdentityState[T]
```

Comment:
Maybe this could be `MaybeDisplayable` instead?

Reply:
I think you mean the other way round? I feel that I'd prefer a name that somehow mentions identifiers, but I agree that the current names are rubbish and can cause confusion, so I'll go for `Displayable` and `MaybeDisplayable` for now